edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/routing/RoutingEdgeHub.cs

```diff
@@ -57,9 +57,13 @@ public Task ProcessDeviceMessage(IIdentity identity, IMessage message)
 
         public Task ProcessDeviceMessageBatch(IIdentity identity, IEnumerable<IMessage> messages)
         {
-            IEnumerable<IRoutingMessage> routingMessages = Preconditions.CheckNotNull(messages)
-                .Select(m => this.ProcessMessageInternal(m, true));
-            return this.router.RouteAsync(routingMessages);
+            Metrics.MessageCount(identity);
```

Comment:
You are incrementing by 1, is this correct? Also, won't this measure as if the latency is for 1 message?

Reply:
Good catch. Changed.